test/unit/host_test.rb

```diff
@@ -912,8 +912,14 @@ def teardown
   end
 
   test "should have only one bootable interface" do
-    subnet = FactoryGirl.create(:subnet)
-    h = FactoryGirl.create(:host, :managed, :subnet => subnet, :ip => subnet.network.succ)
+    organization = FactoryGirl.create(:organization)
+    location = FactoryGirl.create(:location)
+    subnet = FactoryGirl.create(:subnet, :organizations => [organization], :locations => [location])
+    h = FactoryGirl.create(:host, :managed,
```

Comment:
Use `FactoryGirl.build` if possible instead, it avoids hitting the database and makes the test faster.